frontend/src/main/web/lib/stores/UserMatrixStore.js

```diff
@@ -217,27 +219,29 @@ var UserMatrixStore = assign({}, EventEmitter.prototype, {
   dispatchToken: Dispatcher.register(function(payload) {
     var action = payload.action;
     switch (action['actionType']) {
-      case ActionTypes.DATE_RANGE_UPDATE:
+      case UserMatrixActionTypes.DATE_RANGE_UPDATE:
         console.log('date range from %s -> %s', _state['dateRangeOption'], action.data);
         _state['dateRangeOption'] = action.data;
         _state['selectedDay'] = null;
-        loadFromServer()
-          .then(handleServerResponse)
-          .then(function(newState) {
-            UserMatrixStore.emitChange();
-          })
-          .catch(function(err) {
-            console.error('something bad happen:' + err.stack);
-          });
+        if(Configs.user.loggedIn) {
+          loadFromServer()
+            .then(handleServerResponse)
+            .then(function(newState) {
+              UserMatrixStore.emitChange();
+            })
+            .catch(function(err) {
+              console.error('something bad happen:' + err.stack);
```

Comment:
Could provide context of what was happening: `'failed trying to load <whatever this is> from server'`